PumaRhino/utils.h

```diff
@@ -113,7 +113,7 @@ std::string toUTF8FromUTF16(const std::wstring& utf16String);
 std::string toUTF8FromOSNarrow(const std::string& osString);
 
 void appendToRhinoString(ON_wString& rhinoString, const std::wstring& appendee);
-void appendColor(const ON_Color& color, ON_SimpleArray<int>* pArray);
+void appendColor(const ON_Color& color, ON_SimpleArray<double>* pArray);
```

Comment:
This also needs to be reconsidered once we have proper integer wrappers (but I still would prefer to have colors as doubles in the 0..1 range all the way from GH to PRT).

Reply:
I kept these as doubles in the 0..1 range.